app/views/overview/_list-row-actions.html

```diff
@@ -15,7 +15,17 @@
               <a href="" ng-click="row.startBuild(row.buildConfigs[0])">Start Build</a>
             </li>
             <li ng-if="'deploymentconfigs/instantiate' | canI : 'create'" role="menuitem">
-              <a href="" ng-click="row.startDeployment()">Deploy</a>
+              <a
+                href=""
+                ng-if="row.canDeploy()"
+                ng-click="row.startDeployment()">Deploy</a>
+              <a
+                href=""
+                ng-if="!(row.canDeploy())"
+                class="disabled-link"
+                disabled>
```

Comment:
But `disabled` is not normally an attribute on links like it is for buttons, so I'm not sure if screen readers will respect it. If we really want to be correct, I think we should have 

```
role="button" aria-disabled="true" tabindex="-1"
```